templates/password_pages/password_required.html

```diff
@@ -2,10 +2,6 @@
 
 {% load static wagtailmetadata_tags wagtailimages_tags wagtailcore_tags i18n %}
 
-{% if breadcrumb %}
```

Comment:
I tested pages where I had updated the breadcrumb path, and realised that this one would never render the breadcrumb, as it is not in a block - so I've removed it.